cocos/scene-graph/node.ts

```diff
@@ -1900,6 +1911,12 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
      * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
      */
     public _onBatchCreated (dontSyncChildPrefab: boolean): void {
+        if (this._eventMask & ACTIVE_ON) {
```

Comment:
Why need to check it here?

Reply:
For performance consideration, no all node instance need to dispatch `ACTIVE_CHANGED` events, so checking the mask here for the nodes that are used as `tween` targets .

Reply:
Ref: https://github.com/cocos/cocos-engine/pull/17141